server/src/com/thoughtworks/go/server/service/AgentService.java

```diff
@@ -114,8 +114,8 @@ public void sync(Agents agents) {
     }
 
     @Deprecated // used by old code, stop using AgentViewModel
-    public AgentsViewModel agents() {
-        return toAgentViewModels(agentInstances.allAgents());
+    public AgentInstances agents() {
```

Comment:
No one seems to be using this method apart from the old controller which is being deleted as a part of this PR. Why not just get rid of the method?